modules/supported-platforms-for-managed-clusters.adoc

```diff
@@ -0,0 +1,55 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp_far_edge/ztp-deploying-far-edge-clusters-at-scale.adoc
+
+:_content-type: PROCEDURE
+[id="supported-platforms-for-managed-clusters_{context}"]
+= Supported platforms for managed clusters
+
+You can manually specify the platform type for managed clusters if you do not want to set the platform type automatically.
+
+The {ai-full} supports the following platform types:
+
+* `BareMetal`
+* `VSphere`
+* `None`
+
+.Procedure
+
+* Select the platform in the `AgentClusterInstall` custom resource (CR) by changing the value of the `spec.platformType` parameter:
++
+[source,yaml]
+----
+apiVersion: extensions.hive.openshift.io/v1beta1
+kind: AgentClusterInstall
+metadata:
+  name: cluster
+  namespace: cluster
+spec:
+  platformType: VSphere
+----
+
+In some cases, the system automatically sets the platform type:
+
+* The default platform type for multi-node clusters is `BareMetal`.
+* The default platform type for {sno} is `None`.
+* Enabling the `spec.networking.userManagedNetworking` setting without specifying the platform causes the platform value to be `None`.
+
+The `BareMetal` platform type and `userManagedNetworking` parameter value are not compatible.
+
+[NOTE]
+====
+* `VSphere` is not supported for {sno}.
+* Virtual machine (VM) provisioning should be done by the user.
+* Virtual IP address (VIP) DHCP allocation should be configured manually in `VSphere`.
+====
+
+.Verification
+
+* To verify the selected platform in the `AgentClusterInstall` CR by checking the value of the `spec.platformType` parameter, run the following command:
++
+[source,terminal]
+----
+$ oc get AgentClusterInstall cluster -n cluster -o jsonpath='{.status.platformType}{"\n"}'
```

Comment:
Use a backslash to break up multiple lines of code
```suggestion
$ oc get AgentClusterInstall cluster -n cluster -o jsonpath='{.status.platformType}{"\n"}' \
```

Reply:
Updated as suggested.